nessie/src/main/java/org/apache/iceberg/nessie/NessieTableOperations.java

```diff
@@ -135,71 +135,26 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
     boolean newTable = base == null;
     String newMetadataLocation = writeNewMetadataIfRequired(newTable, metadata);
 
-    String refName = client.refName();
-    boolean failure = false;
+    AtomicBoolean failure = new AtomicBoolean(false);
```

Comment:
why does this need to be `AtomicBoolean`?

Reply:
removed `AtomicBoolean` as it can be simplified.